_data/glossary.yaml

```diff
@@ -134,9 +132,9 @@ filesystem: |
 
   Examples :
 
-  - Linux : ext4, aufs, btrfs, zfs
+  - Linux : overlay2, extfs, btrfs, zfs
```

Comment:
`overlay2` is probably wrong in this context, as that's the name of a storage driver in Docker Engine, but the name in the kernel would be `overlayfs` 🤔

Reply:
That‘s my fault 😅

Reply:
no worries! we need to consider what we want to do with the glossary altogether; I guess it's not viewed too often